OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseAnnouncementsFragment.java

```diff
@@ -73,9 +68,9 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
 
-        View view = inflater.inflate(R.layout.fragment_course_combined_info, container, false);
+        View view = inflater.inflate(R.layout.fragment_webview_with_paddings, container, false);
 
-        announcementWebView = (EdxWebView) view.findViewById(R.id.announcement_webview);
+        announcementWebView = view.findViewById(R.id.webview);
```

Comment:
`announcement_webview` variable name should be renamed to `webview`, 'announcementWebView' webview seems odd here.